.eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
   rules: {
     ...baseline.rules,
     'import/prefer-default-export': ['off'],
+    'import/no-cycle': ['error', { ignoreExternal: true }],
```

Comment:
Perhaps we can use an override just like in https://github.com/mui/material-ui/pull/37223/files so that we benefit from the perf boost as well?